.github/workflows/nightly-tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Run (intensive) tests nightly
+on:
+  # todo: remove
+  push:
+  schedule:
+      # 2:15 AM PDT, offseted by a few minutes because:
+      # "The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped."
+      - cron: '15 9 * * *'
+
+jobs:
+  test-distributed:
```

Comment:
nit - rename to test-cluster-slow or test-cluster-nightly or similar (the other job is test-cluster, so just want it consistent, we can also change everything else to distributed instead ofc)